config/optional/views.view.civiremote_funding_combined_application_process_list.yml

```diff
@@ -1,10 +1,14 @@
-langcode: en
+langcode: de
```

Comment:
This should still be `en`.

Reply:
this probably comes from the drupal views export as the site is configured to german language. i will switch language in the exported view back to en